pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = ["Colton Wolkins <colton@indicio.tech>", "Kim Ebert <kim@indicio.tech>
 
 [tool.poetry.dependencies]
 python = "^3.9.0"
-aries-cloudagent = {version = "^0.11.0", extras = ["askar"]}
+aries-cloudagent = {version = "^0.12.1", extras = ["askar"]}
```

Comment:
If no changes are required in ACA-Py cache redis to get this working on 0.12.1, then there's technically no need to change this version constraint in the pyproject.toml. We can just run a `poetry update` and it should update the lock file and that should be all we need.

Reply:
```
Because acapy-cache-redis (0.1.0) @ git+https://github.com/Indicio-tech/aries-acapy-cache-redis.git@HEAD depends on aries-cloudagent (^0.11.0)
 and no versions of aries-cloudagent match >0.11.0,<0.12.0, acapy-cache-redis (0.1.0) @ git+https://github.com/Indicio-tech/aries-acapy-cache-redis.git@HEAD requires aries-cloudagent (0.11.0).
```

Given this error message @anwalker293 reported, this indicates that the version constraint needs to be tweaked. I recommend changing it to `>=0.11.0,<1.0.0`.